Client/Frontend/Reader/ReaderMode.swift

```diff
@@ -36,10 +36,48 @@ enum ReaderModeFontType: String {
 
 enum ReaderModeFontSize: Int {
     case Smallest = 1
-    case Small
+    case Small = 2
     case Normal = 3
     case Large = 4
     case Largest = 5
+
+    func isSmallest() -> Bool {
+        return self == Smallest
+    }
+
+    func smaller() -> ReaderModeFontSize {
+        switch self {
+        case Smallest:
```

Comment:
You could probably write this using ReaderModeFontSize(rawValue: self.rawValue - 1), but I guess that would require some min-max clamping as well... Not a big thing.